qa/tasks/workunit.py

```diff
@@ -321,6 +321,7 @@ def _run_tests(ctx, refspec, role, tests, env, subdir=None, timeout=None):
                 run.Raw('&&'),
                 'git',
                 'clone',
+                '--depth=1',
```

Comment:
we can also pass `--no-checkout,` to `git clone`, to save some time to check out the HEAD. we will run `git check` later one anyway.